flags.go

```diff
@@ -29,6 +31,11 @@ type Backend interface {
 	Refresh() (map[string]Flag, error)
 }
 
+type UpdatableBackend interface {
```

Comment:
We should document what this means :)  Also let's try and choose a better name, my initial stab was super confusing

Reply:
Removed this interface...tacked on "age" to the existing Refresh method..let me know if better naming is needed there as well!